File: backend/src/main/java/fr/formation/model/TypePrevention.java
package fr.formation.model;

public enum TypePrevention {
    medical("Médical"), barriere("Barrière"), comportement("Comportemental");

    private String typePrevention;

    private TypePrevention(String typePrevention) {
        this.typePrevention = typePrevention;
    }

    public String getTypePrevention() {
        return typePrevention;
    }

    public void setTypePrevention(String typePrevention) {
        this.typePrevention = typePrevention;
    }

}

File: frontend/src/app/enum/type-prevention.ts
export type TypePrevention = 'medical' | 'barriere' | 'comportement';

export const TypePreventionEmoji: Record<TypePrevention, { emoji: string; label: string }> = {
  medical: { emoji: '💊', label: 'Médical' },
  barriere: { emoji: '🛡️', label: 'Barrière' },
  comportement: { emoji: '👥', label: 'Comportemental' },
};
